Core/OxidEE_Events.php

```diff
@@ -160,38 +157,26 @@ private static function _createOrderTable()
     private static function _createOrderTransactionTable()
     {
         $sQuery = "CREATE TABLE IF NOT EXISTS `wdoxidee_ordertransactions`(
-            `wdoxidee_transactionid` int(11) unsigned NOT NULL AUTO_INCREMENT,
-            `wdoxidee_orderid` char(32) character set latin1 collate latin1_general_ci NOT NULL,
-            `wdoxidee_requestid` varchar(256) NOT NULL,
-            `wdoxidee_transactiontype` varchar(128) NOT NULL,
-            `wdoxidee_amount` decimal(9,2) NOT NULL,
-            `wdoxidee_refundedamount` decimal(9,2) NOT NULL,
-            `wdoxidee_currency` varchar(32) NOT NULL,
-            `wdoxidee_transactiondate` datetime NOT NULL,
-            `wdoxidee_transactionstatus` enum('pending','success','error') NOT NULL DEFAULT 'pending',
-            `wdoxidee_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
-            `wdoxidee_comment` mediumtext,
-            PRIMARY KEY (`wdoxidee_transactionid`),
-            KEY `wdoxidee_orderid` (`wdoxidee_orderid`),
-            KEY `wdoxidee_transactiondate` (`wdoxidee_transactiondate`)
+            `OXID` char(32) NOT NULL,
+            `WDOXIDEE_ORDERID` varchar(32) NOT NULL,
+            `WDOXIDEE_TRANSACTIONID` varchar(32) NOT NULL,
+            `WDOXIDEE_PARENTTRANSACTIONID` varchar(32),
+            `WDOXIDEE_REQUESTID` varchar(32) NOT NULL,
+            `WDOXIDEE_ACTION` enum('purchase','authorization') NOT NULL DEFAULT 'purchase',
```

Comment:
Is not needed in the transaction overview, so i see it as optional. Mandatory is Transaction type(debit, capture-authorization, authorization,...)

Reply:
We added it because of the OpenCart screenshot attached to the ticket. 

https://jira.parkside.at/browse/WDPD-27

Reply:
This is ok, but don't get confused, this is the transaction type returned with the response. Not the action that is set in the back end for a payment method. So this can still be debit, capture-authorization... Is that taken in consideration?

Reply:
Yes, it is.